src/style.css

```diff
@@ -0,0 +1,149 @@
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgb(6, 11, 53);
+  margin-bottom: 4px;
+}
+
+.card__quote {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  align-self: flex-start;
+  margin-bottom: 16px;
+}
+
+.card__raiting,
+.card__price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__raiting {
+  margin-bottom: 24px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.card__comment {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgb(6, 11, 53);
+}
+
+.card__price-description {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: rgb(6, 11, 53);
+}
+
+.card__linc {
+  background: rgb(0, 172, 220);
+  color: rgb(255, 255, 255);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__linc--hover:hover {
+  background: rgb(255, 255, 255);
+  transition: hover 0.4s linear;
+  color: rgb(0, 172, 220);
+  border: 1px solid rgb(0, 172, 220);
+}
```

Comment:
you don't need an especial `--hover` modification for `.card__link`, you don't use `.card__link` itself without `--hover` 😉

Reply:
combine this 2 rules into one